build/k8s/install-playbook/pop-env.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export BOOTSTRAP="/dns4/bootstrap.myel.cloud/tcp/4001/ipfs/12D3KooWML7NMZudk8H4v1AptitsTZdDqLKgEzoAdLUwuKPqkLyy"
+export FIL_ENDPOINT="https://lotus.myel.cloud/rpc/v0"
+export FIL_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkIl19.nrb3kduJKHEsjl0qojG6umCLRzml_1BZCPxESrHIQ8g"
```

Comment:
@alexander-camuto this shouldn't work since we're using Infura. Also is there a way to avoid committing the token in our repo? Just don't want it to be public.

Reply:
Actually do we even need a token if we use the Infura proxy?

Reply:
probably no longer need a token in the pop setup tbh